packages/api-client/src/tcp/ReconnectingWebsocket.ts

```diff
@@ -121,7 +121,7 @@ export class ReconnectingWebsocket {
   };
 
   private readonly internalOnClose = (event: CloseEvent) => {
-    this.logger.debug('WebSocket closed', event);
+    this.logger.debug('WebSocket closed');
```

Comment:
The data in the event is quite nice because it includes the close reason for example

Reply:
Yes but it spams the terminal with a lot of unnecessary data.

Reply:
How about setting the debug level to info instead of debug?

Reply:
Please point me to the part of logdown where I can set the debug level. :wink:

Reply:
I agree that we should remove the `event` from the log. It is a complex object which does not look good when being stringified. If someone is interested in that object the debugger in Chrome (or attached to Node.js) can be used.

Reply:
![image](https://user-images.githubusercontent.com/1138344/72437347-d4a47a00-37a2-11ea-848b-866798969763.png)

Reply:
That's the Console level selector in Chrome, I was talking about terminals :)